Plugins/Encounter/Encounter.cpp

```diff
@@ -278,4 +281,95 @@ ArgumentStack Encounter::GetCurrentNumSpawned(ArgumentStack&& args)
     return Services::Events::Arguments(retVal);
 }
 
+ArgumentStack Encounter::GetEncounterGeometry(ArgumentStack&& args)
+{
+    std::string retVal;
+
+    if (auto *pObject = encounter(args))
+    {
+        if (auto *pEncounter = Utils::AsNWSEncounter(pObject))
+        {
+            retVal.reserve(32 * pEncounter->m_nNumActivateVertices);
+
+            for(int i = 0; i < pEncounter->m_nNumActivateVertices; i++)
+            {
+                retVal += "{" + std::to_string(pEncounter->m_pvActivateVertices[i].x) + ", " +
+                                std::to_string(pEncounter->m_pvActivateVertices[i].y) + ", " +
+                                std::to_string(pEncounter->m_pvActivateVertices[i].z) + "}";
+            }
+        }
+        else
+        {
+            LOG_WARNING("NWNX_Encounter_GetEncounterGeometry() called on non encounter object.");
```

Comment:
This else branch is not needed, `encounter()` function will print the warning already. Same in the other function.